Core.lua

```diff
@@ -802,7 +804,7 @@ local function BuildGroupieWindow()
 
     local OtherTabButton = CreateFrame("Button", "GroupieTab8", GroupieFrame, "CharacterFrameTabButtonTemplate")
     OtherTabButton:SetPoint("LEFT", "GroupieTab7", "RIGHT", -16, 0)
-    OtherTabButton:SetText("Other")
+    OtherTabButton:SetText(L["Other"])
```

Comment:
```suggestion
    OtherTabButton:SetText(_G.OTHER)
```

https://github.com/tekkub/wow-globalstrings/blob/master/GlobalStrings/enUS.lua#L1040